test/shell/move-upper-dir.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+# TEST FOR #124 --- Disabled for now
```

Comment:
Can you add a FIXME: prefix here so scripts I use to find issues will see this please?